addons/website_forum/tests/test_performance.py

```diff
@@ -0,0 +1,71 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import random
+
+from odoo.addons.website.tests.test_performance import UtilPerf
+
+
+class TestForumPerformance(UtilPerf):
+    def setUp(self):
```

Comment:
Use setUpClass, no need to create data for each test

Reply:
Good to know thank you.